diffpy/snmf/tests/test_subroutines.py

```diff
@@ -134,3 +134,20 @@ def test_reconstruct_data(trd):
     actual = reconstruct_data(trd[0][0], trd[0][1], trd[0][2], trd[0][3], trd[0][4], trd[0][5])
     expected = trd[1]
     np.testing.assert_allclose(actual, expected, rtol=1e-03)
+
+
+tld = [([[[1, -1, 1], [0, 0, 0], [2, 10, -3]], 1], [[4, 2, 4], [3, 3, 3], [5, 13, 0]]),
+       ([[[1, -1, 1], [0, 0, 0], [2, 10, -3]], 0], [[1, -1, 1], [0, 0, 0], [2, 10, -3]]),
+       ([[[1, -1, 1], [0, 0, 0], [2, 10, -3]], .5], [[2.5, .5, 2.5], [1.5, 1.5, 1.5], [3.5, 11.5, -1.5]]),
+       ([[[1, -1, 1], [0, 0, 0], [2, 10, -3]], -1], [[-2, -4, -2], [-3, -3, -3], [-1, 7, -6]]),
+       ([[[0, 0, 0], [0, 0, 0], [0, 0, 0]], 100], [[0, 0, 0], [0, 0, 0], [0, 0, 0]]),
+       ([[[1.5, 2], [10.5, 1], [0.5, 2]], 1], [[1, 1.5], [10, .5], [0, 1.5]]),
+       ([[[-10, -10.5], [-12.2, -12.2], [0, 0]], 1], [[2.2, 1.7], [0, 0], [12.2, 12.2]]),
+       ]
+
```

Comment:
please remove empty lines here

Reply:
I will. Pycharm's autoformatter is what is adding the lines

Reply:
Turn it off....    We can lint later to make everything pep8 compliant if we want.